friends_3ds.go

```diff
@@ -228,7 +299,7 @@ type FriendPersistentInfo struct {
 	Message      string
 	MsgUpdatedAt *nex.DateTime //appears to be correct, but not 100% sure.
```

Comment:
I know you didn't write this line but `MsgUpdatedAt` should probably be changed to `MessageUpdatedAt` for consistency purposes while you're here